web-ui/src/App.jsx

```diff
@@ -10,66 +10,79 @@ import { AppContextProvider } from './context/AppContext';
 import SnackBarWithContext from './components/snackbar/SnackBarWithContext';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 import { LocalizationProvider } from '@mui/x-date-pickers';
-import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+import {
+  experimental_extendTheme as extendTheme,
```

Comment:
What is the significance of starting names with "experimental"?

Reply:
It was just what the docs said to use these features.